doc/sphinx-guides/source/api/client-libraries.rst

```diff
@@ -44,12 +44,14 @@ There is no official PHP library for Dataverse APIs (please :ref:`get in touch <
 Python
 ------
 
-There are two Python modules for interacting with Dataverse APIs.
+There are three Python packages for interacting with Dataverse APIs.
```

Comment:
```suggestion
There are multiple Python packages for interacting with Dataverse APIs.
```

"Multiple" is probably better because any day @JR-1991 will add easyDataverse! 😄